doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -766,58 +726,81 @@ the conversion in hypotheses :n:`{+ @ident}`.
          pattern (0 = 0).
          fold not.
 
-   .. tacv:: fold {+ @term}
+.. tacn:: pattern {+, @pattern_occs } {? @occurrences }
 
-      Equivalent to :n:`fold @term ; ... ; fold @term`.
+   .. todo PR: how do multiple pattern_occs work with @occurrences??
 
-.. tacn:: pattern @term
-   :name: pattern
+   Performs :term:`eta-expansion`.  If :n:`@occurrences` is given,
+   the specified occurrences will be replaced in the selected hypotheses and/or goal.
+   If :n:`@occurrences` is not given, every occurrence of the resulting term in the goal is
+   of the current goal (say :g:`T`)
+   for a set of :n:`@one_term`\s in the :n:`@pattern_occs` by:
```

Comment:
The grammar here is wrong.

Reply:
Can you be more specific?

Reply:
"every ... in the goal is of the current goal for a set of ... by:" seems very strange, in particular I don't know what it means for a term/occurence to be "of the current goal", and I'm confused how "by" is supposed to modify the sentence

Reply:
Oh. I thought you meant Coq's grammar, not English grammar.